lib/rules/object-shorthand.js

```diff
@@ -100,6 +103,7 @@ module.exports = {
         const PARAMS = context.options[1] || {};
         const IGNORE_CONSTRUCTORS = PARAMS.ignoreConstructors;
         const AVOID_QUOTES = PARAMS.avoidQuotes;
+        const ALLOW_EXPLICIT_RETURN_ARROWS = typeof PARAMS.allowExplicitReturnArrows === "undefined" || PARAMS.allowExplicitReturnArrows;
```

Comment:
I feel that boolean options should do the same thing when omitted as when `false` is provided - thus, no options should ever default to `true`.

I'd love to see an option name here that was inverted, allowing it to default to `false` - perhaps `disallowExplicitReturnArrows`?